### PR TITLE
Release/7.0.1

### PR DESCRIPTION
**Pull request by PhungNg**
# Changelog

## Changed
- Content on Links page

## Fixed
- Screen reader reading the bubble images on the home page
- Border radius bug on a:focus
- Topbar missing focus state
- Input field > Autocomplete section not showing "autocomplete"-attribute
- Content in component overview not centered when option menu was closed
- A lot of links not pointing to the right direction

- Safari bugs
  - Line breaks and text ends up outside of tooltip
  - Tabs scrollable shadow
  - Help icon for checkbox and radio button was not align correct
  - Default checkbox showed up on Togglebox

### CODE REVIEW

File: src/App/ComponentsDocumentation/components/InputField/constants.js [hunk 1]
```diff
@@ -174,7 +174,7 @@ export const textareaShowCase = {
                 ]
             },
             title: "Textarea",
-            description: "Textareas have a few different sates which includes the default state, focus state, disabled state and error state. "
+            description: "Text areas have a few different sates which includes the default state, focus state, disabled state and error state. "
```
Comment (VikingTristan) [inline src/App/ComponentsDocumentation/components/InputField/constants.js:177]: imo
```suggestion
            description: "Textareas have a few different sates which includes the default state, focus state, disabled state and error state. "
```

File: src/App/ComponentsDocumentation/components/InputField/index.js [hunk 2]
```diff
@@ -32,77 +30,74 @@ const Textarea = () => (
...
-                <div className="slab slab-plain slab-border-success pb-4">
-                    {/* <span className="h3 mb-3">Do</span> */}
-                    <h3 className="mb-3 mt-0">Do</h3>
-                    <InputGroup type="text" label="Personal authentication number" placeholder="YYYYMMDDNNNN"/>
-                </div>
-                <p>Use the placeholder text as an example of what to enter.</p>
+    </div>
+
+    <h3>Input label</h3>
+    <p>A input field must have a label that clearly describes the type of input a field requires. </p>
```
Comment (VikingTristan) [inline src/App/ComponentsDocumentation/components/InputField/index.js:46]:
```suggestion
    <p>An input field must have a label that clearly describes the type of input a field requires. </p>
```

File: src/less/components/forms/checkbox.less [hunk 3]
```diff
@@ -116,14 +116,20 @@
...
-                font-weight: normal;
+            .help-icon {
+                transform: translateY(25%);
+                margin-left: 0;
             }
         }
+
+        span {
+            font-weight: normal;
+        }
```
Comment (VikingTristan) [inline src/less/components/forms/checkbox.less:132]: image

File: src/scripts/main/sidebar/index.js [hunk 4]
```diff
@@ -338,7 +338,7 @@ const populateSidebarTertiary = (id, leafList) => {
         const newLeafContent = document.createElement("a");
 
         newLeafContent.textContent = leaf.textContent;
-        // newLeafContent.href = `#${leaf.id}`;
+        newLeafContent.href = "javascript:;";
```
Comment (VikingTristan) [inline src/scripts/main/sidebar/index.js:341]: This is a little scary and is gonna be a problem for projects with CSP etc

Comment (VikingTristan) [reply src/scripts/main/sidebar/index.js:341]: Wait with this until next "patch" or somehting

**Review by VikingTristan — CHANGES_REQUESTED** [review]
Looks great! Got a couple of minor comments

File: src/scripts/main/sidebar/index.js (hunk 4 above)
Comment (PhungNg) [reply src/scripts/main/sidebar/index.js:341]: This line is now commented out

**Review by VikingTristan — APPROVED** [review]

**Pull request merged**